index.html

```diff
@@ -35,8 +35,95 @@ <h2>I’m a software developer</h2>
+                <li class="l1"><a >Ruby on rails</a></li>
+                <li class="l2"><a >CSS</a></li>
+                <li class="l3"><a >JavaScript</a></li>
+                <li class="l4"><a >html</a></li>
+            </ul>
+            
+            
+        </article>
+        <button type="button" class="see">See projects</button>
+
+    </div>
+    <div class="allw">
+        <article class="work1">
+            <div class="snapshot">
+                <img src="./Assets/Snapshoot.png" alt="">
+            </div>
+            <div>
+                <h3 class="multi">  Multi-Post Stories Gain+Glory</h3>
+            </div>
+            <ul class="lang">
+                <li class="l1"><a >Ruby on rails</a></li>
+                <li class="l2"><a >CSS</a></li>
+                <li class="l3"><a >JavaScript</a></li>
+                <li class="l4"><a >html</a></li>
+            </ul>
+            
+            
+        </article>
+        <button type="button" class="see">See projects</button>
+
+    </div>
+    <article class="aboutme">
+
+        <h1>About Me</h1>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+        see and have a project you need coded,
+        don’t hestiate to contact me.</p>
+                <button type="button" class="getres">Get My Resume</button>
+
+    </article>
+
+    <article class="languages">
+        <img src="./Assets/Icons.png" alt="">
+        <h1>Languages</h1>
+                    <ul class="lang">
+                        <li class="lr"><a>Ruby on rails</a></li>
+                        <li class="lc"><a>CSS</a></li>
+                        <li class="lj"><a>JavaScript</a></li>
+                        <li class="lh"><a>html</a></li>
+                    </ul>
+
+    </article>
+    <article class="frame">
+        <img src="./Assets/Icons (1).png" alt="">
+        <h1>Frameworks</h1>
+                    <ul class="fr">
+                        <li class="lc"><a>Bootstrap</a></li>
+                        <li class="lr"><a>Ruby on rails</a></li>
+                        <li class="lj"><a>RSpec</a></li>
+                        <!-- <li class="lh"><a>Capypara</a></li> -->
+                        <!-- <li class="lh"><a>Selenium</a></li> -->
+                    </ul>
+
+    </article>
+    <article class="frame">
+        <img src="./Assets/Icons (2).png" alt="">
+        <h1>Skills</h1>
+                    <ul class="fr">
+                        <li class="lc"><a>Bootstrap</a></li>
+                        <li class="lr"><a>Ruby on rails</a></li>
+                        <li class="lj"><a>RSpec</a></li>
+                        <li class="lh"><a>Capypara</a></li>
+                        <li class="lh"><a>Selenium</a></li>
+                    </ul>
+
+    </article>
+
+    <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
```

Comment:
[OPTIONAL] I think this is a paragraph and not a header. This way will be semantically correct and the screen readers and other technologies will thank you. Please make sure you use a p tag here